Formula/logstash.rb

```diff
@@ -44,14 +44,22 @@ def install
               /^LOGSTASH_HOME=.*$/,
               "LOGSTASH_HOME=#{libexec}"
 
+    # Delete Windows and other Arch/OS files
+    rm Dir["bin/*.bat"]
+    os = "Darwin"
+    on_linux { os = "x86_64-Linux" }
+    Dir["vendor/jruby/lib/jni/*"].each do |path|
+      rm_r path unless path.include? os
```

Comment:
Not sure if there is better way to handle this. Could use a more specific check like `File.basename(path) == os` or use path methods using `buildpath/"vendor/.../jni/*"`

On side note, I think the library inside Darwin directory is reason why `logstash` is in universal-allowlist.